backend/aws_helpers/dynamo_db_utils/execution_db.py

```diff
@@ -31,4 +36,53 @@ class ExecutionDDBUtil(BaseDDBUtil):
 
 def get_execution_table(region:str = AWS_REGION) -> BaseDDBUtil:
```

Comment:
is `get_execution_table` ever used or referenced anywhere else? if not, then it's not needed